packages/core/admin/admin/src/pages/Home/HomePage.tsx

```diff
@@ -14,21 +14,47 @@ import { useStrapiApp } from '../../features/StrapiApp';
 import { LastEditedWidget, LastPublishedWidget } from './components/ContentManagerWidgets';
 import { GuidedTour } from './components/GuidedTour';
 
+import type { Widget as WidgetType } from 'src/core/apis/Widgets';
+
 /* -------------------------------------------------------------------------------------------------
  * WidgetRoot
  * -----------------------------------------------------------------------------------------------*/
 
 interface RootProps {
   title: MessageDescriptor;
   icon?: typeof import('@strapi/icons').PuzzlePiece;
+  permissions?: WidgetType['permissions'];
   children: React.ReactNode;
 }
 
-export const WidgetRoot = ({ title, icon = PuzzlePiece, children }: RootProps) => {
+export const WidgetRoot = ({
```

Comment:
This component is used in the stable version and the future version. Do the changes here have an impact on the stable version?

Reply:
I made a small tweak to always grant the permissions if the future flag isn't on 👍